hooks/boot/01-helpers.rb

```diff
@@ -0,0 +1,44 @@
+class Kafo::Helpers
```

Comment:
It shouldn't be extending Kafo's modules from here, it should use its own namespace.